numpy/_core/code_generators/ufunc_docstrings.py

```diff
@@ -4885,3 +4885,141 @@ def add_newdoc(place, name, doc):
 
 add_newdoc('numpy._core.umath', '_expandtabs_length', '')
 add_newdoc('numpy._core.umath', '_expandtabs', '')
+
+add_newdoc('numpy._core.umath', '_center',
+    """
+    Return a copy of `x1` with its elements centered in a string of
+    length `x2`.
+
+    Parameters
+    ----------
+    x1 : array_like, with `np.bytes_` or `np.str_` dtype
+
+    x2 : array_like, with any integer dtype
+        The length of the resulting strings
+    x3 : array_like, with `np.bytes_` or `np.str_` dtype
+        The padding character to use.
+        $PARAMS
+
+    Returns
+    -------
+    out : ndarray
+        Output array of str or unicode, depending on input
+        types
+        $OUT_SCALAR_2
+
+    See Also
+    --------
+    str.center
+
+    Examples
+    --------
+    >>> c = np.array(['a1b2','1b2a','b2a1','2a1b']); c
+    array(['a1b2', '1b2a', 'b2a1', '2a1b'], dtype='<U4')
+    >>> np.strings.center(c, width=9)
+    array(['   a1b2  ', '   1b2a  ', '   b2a1  ', '   2a1b  '], dtype='<U9')
+    >>> np.strings.center(c, width=9, fillchar='*')
+    array(['***a1b2**', '***1b2a**', '***b2a1**', '***2a1b**'], dtype='<U9')
+    >>> np.strings.center(c, width=1)
```

Comment:
I think this example at least is actually wrong, right, since in this case the original strings are returned.